cmake/sysbuild/sdp.cmake

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Update FLPR image KConfig based on main app configuration.
+#
+# Usage:
+#   flpr_egpio_update_kconfig()
+#
+function(flpr_egpio_update_kconfig)
+
+  sysbuild_get(CONFIG_MBOX_BACKEND IMAGE ${DEFAULT_IMAGE} VAR CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX KCONFIG)
+  sysbuild_get(CONFIG_ICMSG_BACKEND IMAGE ${DEFAULT_IMAGE} VAR CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG KCONFIG)
+  sysbuild_get(CONFIG_ICBMSG_BACKEND IMAGE ${DEFAULT_IMAGE} VAR CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICBMSG KCONFIG)
```

Comment:
why are these not sysbuild Kconfigs that then get sync'd to the main image and secondary image?

Reply:
@jaz1-nordic can you answer this? It was added in #16592 , I only added the last line.